packages/create-react-app-demo/src/docs-chat/ai.tsx

```diff
@@ -0,0 +1,79 @@
+/** @jsx LLMx.createElement */
+/** @jsxFrag LLMx.Fragment */
+/* eslint-disable react/jsx-key */
+import * as LLMx from 'ai-jsx';
+import React, { useEffect, useRef } from 'react';
+import { DocsQA, ScoredChunk, LocalCorpus, defaultChunker, staticLoader } from 'ai-jsx/batteries/docs';
+import { memo } from 'ai-jsx/core/memoize';
+// @ts-expect-error
+import markdownPath from './brand-new.md';
+import { atom, useAtom } from 'jotai';
+
+import _ from 'lodash';
+
+export class ChatMessage {
+  type: string;
+  content: string;
+  constructor({ type, content }: { type: string; content: string }) {
+    this.type = type;
+    this.content = content;
+  }
+}
```

Comment:
I don't think this needs to be a full-fledged `class`.

```suggestion
export interface ChatMessage {
  type: string;
  content: string;
}
```

Reply:
nit: class -> interface